### DISCUSSION TITLE
Question regarding documentation / measuring units

### DISCUSSION THREAD

**Discussion by Niss-Ener** [StefaE/PVForecast#28] [Q&A] [answered]
Hi! 

First of all - thank you very much for this tool! Flexible and easy to use - great job :)

I have a (beginner- ) question regarding the documentation you maybe can help me with: I have difficulties interpreting the values "dc_disc" and "ac_disc" when using SQLite in combination with DWD weather data: 

![grafik](https://github.com/StefaE/PVForecast/assets/162121903/3567dac8-21a5-4b17-af7d-1c81d2f98584)

Unfortunatelly I could not find more information within the documentation (Or I simply did not see it...) Would you mind telling me what measuring units are behind those values?

**Comment by StefaE** [comment, marked as answer]
Hi Niss-Ener

if you work with any of the forecast providers except Solcast, you get a GHI (global horizontal irradiation) value, saying how much solar energy will fall on 1m2 of horizontal surface. This energy is the sum of direct irradiation (DNI) and diffuse (eg. through clouds) brightness (DHI).  Your solar panel 'reacts' on DNI and DHI in different ways.

So - now you have one GHI value, which you need decompose into the two contributors, DNI and DHI. Of course, that can't be done 'exact'; it is the same as asking _which two numbers sum up to the value of five?_ ... There are [different models](https://github.com/StefaE/PVForecast?tab=readme-ov-file#convert-weather-data-to-irradiation-data) to do that however, DISC being the most prominent one (differences in result are negligible for most practical purposes).

Then, you have to convert the result (DNI and DHI) into [output power](https://github.com/StefaE/PVForecast?tab=readme-ov-file#convert-irradiation-data-to-pv-output-power) of your solar system.

Your solar system outputs DC power (dc_disc) which is then converted by your inverter to AC power (ac_disc). Although you can configure different inverters, the rule-of-thumb is that AC = 97%*DC (and 3% are lost in the inverter, give or take).

Hope this clarifies?
Stefan

**Reply by Niss-Ener** [reply]
Yes, thank you very much!
